### PR TITLE
[4.0] Vanilla JS for Aurora template

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue # .

### Summary of Changes

This converts the good ol' jQuery in the Aurora template to vanilla JS.

Also removes the old JS files from both templates that are no longer used.

### CODE REVIEW

**Comment by C-Lodder** [comment]
@brianteeman not sure why you're unhappy with the changes. This is a performance booster and the JS that deals with button classes is from the Atum template. There's absolutely no reason to use jQuery for this, especially when Atum doesn't

File: templates/aurora/js/template.js [hunk 1]
```diff
@@ -1,60 +1,153 @@
...
-		$('fieldset.btn-group').each(function() {
-			// Handle disabled, prevent clicks on the container, and add disabled style to each button
-			if ($(this).prop('disabled')) {
-				$(this).css('pointer-events', 'none').off('click');
-				$(this).find('.btn').addClass('disabled');
-			}
+		/**
+		 * Bootstrap tooltips
+		 */
+		jQuery('*[rel=tooltip]').tooltip({
```
Comment (laoneo) [inline templates/aurora/js/template.js]: Did this get forgotten?

File: templates/aurora/js/template.js [hunk 2]
```diff
@@ -1,60 +1,153 @@
...
+			});
+		}
+
+		var btsGrouped = document.querySelectorAll('.btn-group input[checked=checked]');
+		for (var i = 0, l = btsGrouped.length; l>i; i++) {
+			var self   = btsGrouped[i],
+			    attrId = self.id;
+			if (self.parentNode.parentNode.classList.contains('btn-group-reversed')) {
+				if (self.value == '') {
+					var aa = document.querySelector('label[for=' + attrId + ']');
```
Comment (wilsonge) [inline templates/aurora/js/template.js]: Please can we have something more descriptive than `aa` as a variable name. This is totally unclear

File: templates/aurora/js/template.js (hunk 1 above)
Comment (C-Lodder) [inline templates/aurora/js/template.js]: forgotten?

File: templates/aurora/js/template.js (hunk 2 above)
Comment (C-Lodder) [inline templates/aurora/js/template.js]: will sort this later on

**Comment by C-Lodder** [comment]
@wilsonge - variable names updated

**Pull request merged**